Makefile

```
@@ -1,6 +1,6 @@
 # Reflecta プロジェクト便利コマンド
 
-.PHONY: help dev prod stop prisma-studio chroma-reset chroma-inspect test
+.PHONY: help dev prod stop prisma-studio chroma-reset chroma-inspect neo4j-inspect test
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)